### ISSUE TITLE
Trouble with timeouts

### ISSUE THREAD

**Issue by gaggle** [danielstjules/mocha.parallel#13]
Hi,

I have trouble with timeouts. I've looked at the closed issues that relate to timeouts but didn't get a sense about the specific issue I'm running into. If I run this example:

```
parallel("foo", function () {
  var its = ["bar", "baz"]
  its.forEach(function (n) {
    it(n, function () {
      return new Promise(function (resolve) {
        setTimeout(resolve, 2500)
      })
    })
  })
})
```

I would expect two failing tests as they're both above the default 2000ms timeout, but I actually see one failing and one passing:
```
  foo
    1) bar
    ✓ baz (498ms)


  1 passing (3s)
  1 failing

  1) foo bar:
     Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.
```
Why is that?

I'm actually trying to achieve long timeouts for all tests to run multi-minute integration tests in parallel, but no usage of `this.timeout(300000)` and similar commands altered the above behaviour.

```
npm list --depth=0
+-- mocha@2.4.5
+-- mocha.parallel@0.12.0
```

**Comment by danielstjules** [comment]
Sorry for the late reply! Unfortunately, there's not much I can do without upstream changes https://github.com/danielstjules/mocha.parallel/issues/5#issuecomment-152943025

**Comment by gaggle** [comment]
Ah okay, cheers. Feel free to close this issue as you see fit.

**Comment by danielstjules** [comment]
I'll keep this open for now so other people see it. I may open a PR upstream to help with it as well.

**Comment by danielstjules** [comment]
Fixed and published in 0.13!
